### ISSUE TITLE
vp to vpts with grep, awk, ...

### ISSUE THREAD

**Issue by peterdesmet** [enram/vp-parser#14]
What would be the `grep`, `awk`, `cat` function to concatenate multiple vp files to a vpts file **ignoring all the subsequent headers**? See the example files at https://github.com/enram/vp-parser/tree/main/data ping @adokter

**Comment by peterdesmet** [comment]
grep -v '^#' bewid_vp_20201017T094500Z_0x9.csv

Will ignore all comment lines

**Comment by adokter** [comment]
Command line concatenation I use often is `cat *.csv > my_vpts.csv`.

Or when using vol2bird's stdout for processing multiple h5 volume files: `find *.h5 | awk '{system("vol2bird "$0)}' >> my_vpts.csv`

**Comment by peterdesmet** [comment]
@adokter indeed, but we need a command to ignore the consecutive headers when concatenating.

**Comment by niconoe** [comment]
@peterdesmet: I think it will be easier answer the question once we more clearly define the format (= answer issues: https://github.com/enram/vp-parser/issues?q=is%3Aopen+is%3Aissue+label%3A%22file+format%22.

After that, we can come with:

- either a `grep`, `awk`, `cat`, ... solution as requested
- either build a custom tool to do that. I wouldn't automatically reject this option since 1) it can be pretty simple (if we have properly defined the file format beforehand) 2) it would be reusable (if we have properly defined the file format beforehand) 3) it could provide more robustness and advances features (check data columns actually match, merge consecutive headers into a single one, ...)
